framework/src/modules/chain/blocks/schema.js

```diff
@@ -54,15 +54,15 @@ const baseBlockSchema = {
 			type: 'integer',
 		},
 		totalAmount: {
-			type: 'object',
+			type: ['string', 'object'],
```

Comment:
NOt sure why we expect or string or object? Is it not just a string?

Reply:
This is due to the fact that the function responsible to validate `amount` format accepts both object or string, therefore if only setting the type of `totalAmount` to object, when passing a `string` it was failing, so now it is actually correct and compliant with the format definition.

Reply:
As mentioned above